plugins/discovery-ec2/src/internalClusterTest/java/org/elasticsearch/discovery/ec2/DiscoveryEc2RegularNetworkAddressesIT.java

```diff
@@ -44,7 +44,7 @@ public void testImdsNotAvailable() throws IOException {
             final var bindTransportException = asInstanceOf(BindTransportException.class, executionException.getCause());
             assertEquals("Failed to resolve publish address", bindTransportException.getMessage());
             final var ioException = asInstanceOf(IOException.class, bindTransportException.getCause());
-            assertThat(ioException.getMessage(), containsString("/latest/meta-data/local-ipv4"));
+            assertThat(ioException.getMessage(), containsString("/transport/latest/meta-data/local-ipv4"));
```

Comment:
Is this intentional, or a result of IJ's accidental refactoring of strings?

Reply:
Sigh, accidental :facepalm: